sentry-android-core/src/main/java/io/sentry/android/core/IBuildInfoProvider.java

```diff
@@ -0,0 +1,12 @@
+package io.sentry.android.core;
+
+/** To make the Build class testable */
+public interface IBuildInfoProvider {
```

Comment:
@philipphofmann `final` yes, `static` no, only with `PowerMock` but it's too magic IMO

Reply:
We only want to mock `getSdkInfoVersion` right? If `BuildInfoProvider` isn't a singleton then we can also mock it I guess. I wouldn't create the interface because of YAGNI.

Reply:
@philipphofmann as stated in other PR, this is gonna be reused, so a new method is coming along, eg `Build.TAGS`

can you show me how to mock this with mockito?